src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +89,32 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    /**
+     * @return the blockNumber
+     */
+    public Block getBlockNumber() {
```

Comment:
Wouldn't it have been better to update the toString() to return the whole address from the Address.java class rather than returning individual components?